new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDomainOnPV.java

```diff
@@ -0,0 +1,682 @@
+import static oracle.weblogic.kubernetes.actions.ActionConstants.WLS_BASE_IMAGE_NAME;
+import static oracle.weblogic.kubernetes.actions.ActionConstants.WLS_BASE_IMAGE_TAG;
+import static oracle.weblogic.kubernetes.actions.TestActions.createConfigMap;
+import static oracle.weblogic.kubernetes.actions.TestActions.createDockerConfigJson;
+import static oracle.weblogic.kubernetes.actions.TestActions.createDomainCustomResource;
+import static oracle.weblogic.kubernetes.actions.TestActions.createNamespacedJob;
+import static oracle.weblogic.kubernetes.actions.TestActions.createPersistentVolume;
+import static oracle.weblogic.kubernetes.actions.TestActions.createPersistentVolumeClaim;
+import static oracle.weblogic.kubernetes.actions.TestActions.createSecret;
+import static oracle.weblogic.kubernetes.actions.TestActions.createServiceAccount;
+import static oracle.weblogic.kubernetes.actions.TestActions.getOperatorImageName;
+import static oracle.weblogic.kubernetes.actions.TestActions.getServiceNodePort;
+import static oracle.weblogic.kubernetes.actions.TestActions.installOperator;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.domainExists;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.isHelmReleaseDeployed;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.jobCompleted;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.operatorIsReady;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.podReady;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.serviceExists;
+import static oracle.weblogic.kubernetes.extensions.LoggedTest.logger;
+import static org.awaitility.Awaitility.with;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Tests to create domain on persistent volume using WLST and WDT.
+ */
+@DisplayName("Verify the domain can be created on persistent volume")
+@IntegrationTest
+public class ItDomainOnPV implements LoggedTest {
+
+  private static String opNamespace = null;
+  private static String domainNamespace = null;
+
+  private final String domainUid = "domain-onpv";
+  private final String clusterName = "pv-domain-cluster";
+  private final String adminServerName = "admin-server";
+  private final String adminServerPodName = domainUid + "-" + adminServerName;
+  private final String managedServerNameBase = "managed-server";
+  String managedServerPodNamePrefix = domainUid + "-" + managedServerNameBase;
+
+  private final int replicaCount = 2;
+
+  private final String pvName = domainUid + "-pv"; // name of the persistent volume
+  private final String pvcName = domainUid + "-pvc"; // name of the persistent volume claim
+
+  private String wlSecretName;
+  private final String adminUser = "system";
```

Comment:
i prefer to be consistent - weblogic/welcome1 - helps if we need to get into the domain to debug something, plus it is clearly not real credentials, which helps with secure coding standards compliance

Reply:
changed to weblogic/welcome1